lisa/tools/lscpu.py

```diff
@@ -15,6 +15,19 @@
 )
 
 
+class CPUNodeInfo:
+    def __init__(
+        self, cpu: int, node: int, socket: int, l1i: int, l1d: int, l2: int, l3: int
+    ) -> None:
+        self.cpu = cpu
+        self.node = node
+        self.socket = socket
+        self.l1i = l1i
```

Comment:
the name is not that straightforward.

Reply:
There are two L1 cache : instruction and data. How about I rename it to `l1instruction`?

Reply:
l1_instruction and l1_data?